CHANGELOG/OWNERS

```diff
@@ -0,0 +1,11 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - sig-release-approvers
+  - release-engineering-approvers
+reviewers:
+  - release-engineering-reviewers
```

Comment:
probably also docs reviewers/approvers?

Reply:
Makes sense, but I don't think it's strictly necessary. 
I'll leave it to SIG Docs to make the call on whether they'd like to be added.
cc: @kubernetes/sig-docs-en-owners

Reply:
What's your timeline on approval? I'd love to discuss this with SIG Docs at next Tuesday's weekly meeting, if that's OK.